src/en/pages/index.md

```diff
@@ -60,31 +60,47 @@ date: 'git Last Modified'
 <article class="py-600 bg-light bg-full-width">
   <h2 class="mb-300">What's new</h2>
   <gcds-grid tag="ul" columns="1fr" columns-tablet="1fr 1fr">
-    <li class="list-none bg-white p-450 b-radius-md">
-      <h3 class="mb-300">
-        <gcds-link href="{{ links.getInvolved }}">Get involved</gcds-link>
-      </h3>
-      <p>Check out what we're working on.</p>
-    </li>
-    <li class="list-none bg-white px-175 py-450 b-radius-md">
-      <h3 class="mb-300">
-        <gcds-link external href="{{ links.releaseNotes }}">Release notes</gcds-link>
-      </h3>
-      <p>Read about the latest additions.</p>
-    </li>
+    <gcds-card
+      href="{{ links.registerDemo }}"
+      card-title="Attend a demo"
+      card-title-tag="h3"
+      description="Learn how GC Design System can work for you and your team and get your questions answered."
+      role="listitem"
+    ></gcds-card>
+    <gcds-card
+      href="{{ links.accessibility }}"
+      card-title="Find out about accessibility"
+      card-title-tag="h3"
+      description="Review accessibility practices and checks within the design system."
+      role="listitem"
+    ></gcds-card>
+    <gcds-card
+      href="{{ links.getInvolved }}"
+      card-title="Get involved"
+      card-title-tag="h3"
+      description="Check out what the team’s working on and how you can contribute."
+      role="listitem"
+    ></gcds-card>
+    <gcds-card
+      href="{{ links.releaseNotes }}"
+      card-title="Catch up on releases"
+      card-title-tag="h3"
+      description="Read about the latest additions and features."
+      role="listitem"
+    ></gcds-card>
   </gcds-grid>
 </article>
 
 <article class="py-450">
-  <h2 class="mb-300">Featured component</h2>
+  <h2 class="mb-300">Featured components</h2>
```

Comment:
@SmartMouthWords @ethanWallace I know this design is not new but should we use our card component here instead of plain text? It saves some space and makes the page shorter.

Reply:
Just going to check if we want to do this change